RIAssigner/__main__.py

```diff
@@ -0,0 +1,50 @@
+from RIAssigner.compute.ComputationMethod import ComputationMethod
+import argparse
+
+from RIAssigner.cli import CreateMethodAction, LoadDataAction
+from RIAssigner.data import Data
+
+
+def create_parser():
+    parser = argparse.ArgumentParser(prog="RIAssigner")
+    parser.add_argument("reference",
+                        type=str,
+                        action=LoadDataAction,
+                        help="Reference dataset containing retention times and indices. Path to CSV or MSP.")
+    parser.add_argument("query",
+                        type=str,
+                        action=LoadDataAction,
+                        help="Query dataset for which to compute retention indices. Path to CSV or MSP.")
+    parser.add_argument("method",
+                        type=str,
+                        action=CreateMethodAction,
+                        choices=['kovats', 'cubiscpline'],
+                        help="Computation method for RI.")
+    parser.add_argument("output",
+                        type=str,
+                        help="Output filename.")
+    return parser

```

Comment:
I would personally use [flagged args](https://stackoverflow.com/a/41747010) instead of positional. However, they are generally treated as optional by default. But it can be easily [fixed](https://stackoverflow.com/a/41747010), then the tool prints out nice help and you don't have to specify args in particular order (real world [example](https://github.com/sybila/eBCSgen/blob/d0218e2c7daf43630763a7b2b8d4fdbd07e2dcec/Callables/GenerateTS.py#L14)).